index.js

```diff
@@ -1057,12 +1057,22 @@
   //. of the first occurrence of the value in the list, if applicable;
   //. Nothing otherwise.
   //.
+  //. Dispatches to its second argument's `indexOf` method if present.
+  //. As a result, `String -> String -> Maybe Number` is an alternative
+  //. type signature.
+  //.
```

Comment:
Could a tag work somehow? The obvious benefit is being able to parse those programmatically and show them in docs in some special way if so preferred.

By the way, what's the significance of dot after `//`, if any?